salt/utils/pkg/win.py

```diff
@@ -1,16 +1,3 @@
-# Copyright 2017 Damon Atkins
```

Comment:
Can we remove Copyright information here, given terms of License Apache 2.0, not sure we can pull out Damon's Copyright statement, just thinking back to Berkely vs ATT copyright issue of the 90's when ATT removed BSD copyright statement, the only thing they needed to leave in the code, no matter how much they modified the code.